guide/sapphire/args.md

```diff
@@ -0,0 +1,163 @@
+# Using arguments in commands
+
+Sometimes when using commands, you may want to get data from the user and change the response accordingly. In this section, you'll create a command that pulls a string from the message and says it back to the user!
+
+## String arguments
+
+A `string` argument is simply the text after the command name and prefix. For example: `@bot say Hello!` would cause the argument to be `Hello!`, although multiple words (e.g. `Hello there!`) are separated in different parameters. We'll look into how to handle those later.
+
+First, go into your `commands` folder and make a new file called `say.js`. Once you have it, set up your command class just like the one in the `ping` command.
+
+```js
+const { Command } = require('@sapphire/framework');
+
+module.exports = class SayCommand extends Command {
+	constructor(context) {
+		super(context, {
+			aliases: ['parrot', 'copy'],
+			description: 'Replies with the text you provide',
+		});
+	}
+
+	run(message, args) {
+		// ...
+	}
+};
+```
+
+We have a new parameter in the `run` method: `args`! This is a structure that abstracts all the parameter parsing and resolution.
+
+```js {5}
+module.exports = class SayCommand extends Command {
+	// ...
+
+	async run(message, args) {
+		const text = await args.pick('string');
+		await message.channel.send(text);
+	}
+};
+```
+
+The way Sapphire's arguments work, is by "consuming" the parameters one by one on success, similar to how an iterator works. We'll go into more details and examples with another command.
+
+In this command, we're using `args.pick` to read a single parameter from the user's input. The input type is `string`, which means it'll pick the current parameter and return it as-is.
```

Comment:
`
```suggestion
In this command, we're using `args.pick()` to read a single parameter from the user's input. The input type is `string`, which means it'll pick the current parameter and return it as-is.
```

small change, but would help reinforce the fact this is a method - and makes it easier to locate what you mean in the code example above.